docker-compose.dev.yaml

```diff
@@ -0,0 +1,54 @@
+version: "3"
+
+services:
+  # frontend:
+  #   image: node:14
+  #   volumes:
+  #     - ${PWD}/frontend:/home/mediacms.io/mediacms/frontend/
+  #   working_dir: /home/mediacms.io/mediacms/frontend/
+  #   command: bash -c "npm install && npm run start"
+  #   env_file:
+  #     - ${PWD}/frontend/.env
+  #   ports:
+  #     - "8097:8097"
+  #   depends_on:
+  #     - web
+  web:
```

Comment:
I think you still need to copy the db and redis service into this file. Using the chained `docker-compose -f file1 -f file2` will merge settings from file1 to file2. So the web service will get Environment variables that have intentionally been removed from here (so defaults are used)

Reply:
I don't understand this. So when we run chained command, it inherits non changed values. So won't that bring over redis and db?

Reply:
It will also bring in ENVIRONMENT in web, which has been removed